sycl/source/handler.cpp

```diff
@@ -325,118 +327,118 @@ event handler::finalize() {
     // assert feature to check if kernel uses assertions
     CommandGroup.reset(new detail::CGExecKernel(
-        std::move(MNDRDesc), std::move(MHostKernel), std::move(MKernel),
-        std::move(MImpl->MKernelBundle), std::move(CGData), std::move(MArgs),
-        MKernelName.c_str(), std::move(MStreamStorage),
-        std::move(MImpl->MAuxiliaryResources), MCGType,
-        MImpl->MKernelCacheConfig, MImpl->MKernelIsCooperative,
-        MImpl->MKernelUsesClusterLaunch, MCodeLoc));
+        std::move(impl->MNDRDesc), std::move(MHostKernel), std::move(MKernel),
+        std::move(impl->MKernelBundle), std::move(impl->CGData),
+        std::move(impl->MArgs), MKernelName.c_str(), std::move(MStreamStorage),
+        std::move(impl->MAuxiliaryResources), getType(),
+        impl->MKernelCacheConfig, impl->MKernelIsCooperative,
+        impl->MKernelUsesClusterLaunch, MCodeLoc));
     break;
   }
-  case detail::CG::CopyAccToPtr:
-  case detail::CG::CopyPtrToAcc:
-  case detail::CG::CopyAccToAcc:
+  case detail::CGType::CopyAccToPtr:
+  case detail::CGType::CopyPtrToAcc:
+  case detail::CGType::CopyAccToAcc:
     CommandGroup.reset(
-        new detail::CGCopy(MCGType, MSrcPtr, MDstPtr, std::move(CGData),
-                           std::move(MImpl->MAuxiliaryResources), MCodeLoc));
+        new detail::CGCopy(getType(), MSrcPtr, MDstPtr, std::move(impl->CGData),
+                           std::move(impl->MAuxiliaryResources), MCodeLoc));
     break;
-  case detail::CG::Fill:
+  case detail::CGType::Fill:
     CommandGroup.reset(new detail::CGFill(std::move(MPattern), MDstPtr,
-                                          std::move(CGData), MCodeLoc));
+                                          std::move(impl->CGData), MCodeLoc));
     break;
-  case detail::CG::UpdateHost:
+  case detail::CGType::UpdateHost:
     CommandGroup.reset(
-        new detail::CGUpdateHost(MDstPtr, std::move(CGData), MCodeLoc));
+        new detail::CGUpdateHost(MDstPtr, std::move(impl->CGData), MCodeLoc));
     break;
-  case detail::CG::CopyUSM:
+  case detail::CGType::CopyUSM:
     CommandGroup.reset(new detail::CGCopyUSM(MSrcPtr, MDstPtr, MLength,
-                                             std::move(CGData), MCodeLoc));
+                                             std::move(impl->CGData), MCodeLoc));
     break;
-  case detail::CG::FillUSM:
+  case detail::CGType::FillUSM:
     CommandGroup.reset(new detail::CGFillUSM(
-        std::move(MPattern), MDstPtr, MLength, std::move(CGData), MCodeLoc));
+        std::move(MPattern), MDstPtr, MLength, std::move(impl->CGData), MCodeLoc));
     break;
-  case detail::CG::PrefetchUSM:
+  case detail::CGType::PrefetchUSM:
     CommandGroup.reset(new detail::CGPrefetchUSM(MDstPtr, MLength,
-                                                 std::move(CGData), MCodeLoc));
+                                                 std::move(impl->CGData), MCodeLoc));
```

Comment:
Here and other places are wider than 80 cols. I'm not sure how this wasn't caught by CI